lit_gpt/config.py

```diff
@@ -13,6 +13,20 @@
 from lit_gpt.utils import find_multiple
 
 
+@dataclass
+class BiasMap:
+    main: bool = True
```

Comment:
This is the best what came to my mind.
`global` is reserved variable name.
`all` might mean that all the biases will be with this value, which might not be true (e.g., if we provide a value to mlp).
`default`, `general`, `overall`, ... sounds weird to me.

Any suggestions?